ci-operator/step-registry/baremetalds/assisted/operator/publish/baremetalds-assisted-operator-publish-commands.sh

```diff
@@ -142,14 +157,25 @@ for full_image in $(/tmp/yq eval '.spec.relatedImages[] | .image' "${CO_CSV}");
     image=${full_image%:*}
     registry=${image%%/*}
     image_name=${image#*/}
-    digest=$(curl -G "https://${registry}/api/v1/repository/${image_name}/tag/?specificTag=${tag}" | \
+
+    # Mirror image
+    mirror_image_name="${REGISTRY_ORG}/${image_name#*/}"
+    full_mirror_image="${REGISTRY_HOST}/${mirror_image_name}:${OPERATOR_VERSION}"
+    echo "   mirroring image ${full_image} -> ${full_mirror_image}"
+    oc image mirror "${full_image}" "${full_mirror_image}" || {
```

Comment:
If we decide to pursue this option. We would need to prepopulate (and make public) the quay.io/edge-infrastructure/postgresql-12-centos7 image. To make sure that:
1. We could actually grab the sha
1. end users would be able to pull the image

Reply:
I added this image to the new org in quay.io with a dated tag to pin it just in case:
https://quay.io/repository/edge-infrastructure/postgresql-12-centos7?tab=tags